en/docs/develop/enable-verification-for-updated-mobile-number.md

```diff
@@ -188,6 +207,35 @@ curl -X POST -H "Authorization: Basic Ym9iMTIzOnBhc3MxMjM=" -H "Content-Type: ap
 "HTTP/1.1 201 Created"
 ```
 
+Additionally can send the following curl command to resend a new SMS OTP code by a privileged user.
+
+**Sample**
+
+!!! abstract ""
+    **Request**
+    ```curl
+    curl -X POST -H "Authorization: Basic <Base64Encoded_username:password>" -H "Content-Type: application/json" -d '{"user":{},"properties": []}'
+    "https://localhost:9443/api/identity/user/v1.0/resend-code"
+    ```
+
+    The user and the verification scenario should be specified in the request body as follows :
+    ```
+    "user": {"username": "", "realm": ""}
+    "properties": [{"key":"RecoveryScenario", "value":"MOBILE_VERIFICATION_ON_UPDATE"}]}
+    ```
+    ---
+    **Sample Request**
+    ```curl
+    curl -X POST -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{"user":{"username": "admin","realm": "PRIMARY"},"properties": [{"key":"RecoveryScenario","value":"MOBILE_VERIFICATION_ON_UPDATE"}]}' "https://localhost:9443/api/identity/user/v1.0/resend-code" -k -v
+    ```
+    ---
+    **Response**
+    ```
+    "HTTP/1.1 201 Created"
+    ```
+
+!!! Note
```

Comment:
Seems to be duplicated. The same note was available at the top.
We need to put this notification at the beginning before the user sets out to enable these configurations.

Reply:
Actually this was referring to two updates. Configuration is only required for the mobile number verification. This update is about resending the OTP